test-framework/common/src/main/java/io/quarkus/test/common/RestAssuredURLManager.java

```diff
@@ -70,6 +78,18 @@ public void setURL() {
                 e.printStackTrace();
             }
         }
+        if (basePathField != null) {
+            try {
+                oldBasePath = (String) basePathField.get(null);
+                Optional<String> basePath = ConfigProvider.getConfig().getOptionalValue("quarkus.servlet.context-path",
```

Comment:
Do we need to handle `%test.quarkus.servlet.context-path` too?

Reply:
Well, if I'm right the configuration profile resolution is already done at that point, so I'd answer no.

Reply:
Correct.